cmp/grid/GridModel.js

```diff
@@ -171,13 +171,10 @@ export class GridModel {
     /**
      * Export grid data using Hoist's server-side export.
      *
-     * @param {Object} options
-     * @param {string} options.type - type of export - one of ['excel', 'excelTable', 'csv'].
-     * @param {(string|function)} [options.filename] - name for exported file or closure to generate.
+     * @param {Object} options - Export options. See ExportManager.exportAsync() for options.
      */
     export(options = {}) {
```

Comment:
Any reason not to have this method be exportAsync as well and return the underlying promise?